pkgs/top-level/aliases.nix

```diff
@@ -267,6 +267,7 @@ mapAliases ({
     and has only been used for old fontconfig caches.
   '';
   foomatic_filters = foomatic-filters;  # 2016-08
+  fscryptctl-experimental = throw "The package fscryptctl-experimental has been removed. Please switch to fscryptctl."; # 2021-11-07
```

Comment:
```suggestion
  fscryptctl-experimental = throw "fscryptctl-experimental has been removed. Please switch to fscryptctl."; # 2021-11-07
```

Reply:
Shouldn't really matter but thanks for the review.